### DISCUSSION TITLE
Scrum Prep Milestone 1-3

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#89] [General]
Each team member will comment progress made through the past week.

**Comment by PerezMarcelo** [comment]
This week, I met with team members who had fewer points to discuss their progress and outline necessary tasks. I provided suggestions for task creation, and oversaw their completion. Once tasks met the required criteria, I labeled them as "Lead Review" for final approval.

**Comment by JoshDG03** [comment]
This week I helped implement the Items controller and fill out the needs section of the documentation.

**Comment by CovasJJ** [comment]
This week I worked on documentation. I completed my task of filling out sections 2.1.2 ( Terminology ) and 2.1.3 (Domain terminology related to the rough sketch) of the Documentation.

**Comment by ActuallySeb** [comment]
Wrapping up work for this milestone and focusing on creating a plan for the following one.

-Sebastian Montalvo Nieves

**Comment by JuanDelPueblo** [comment]
This week I completed two tasks. The first one was adding a login page in addition to the sign up page, ensuring that the login would collect the saved account information on the backend and make sure the passwords are the same. The second task was merging the sign up logic branch into main, which required converting our code into Vue in order to be compatible with the web dev team's code.
~ Edyan Cruz

**Comment by DiegoVazquez1** [comment]
This week i implemented the hashing and salting of the users password. I had to do a lot of research with this because i didnt really understand but in the end it worked.  All the code is on the server-side but added something so it would be visible in the client side while we make test.

**Comment by JuanIranzo** [comment]
This week I could not get much work done, however I completed a section of the documentation that focused on the narrative of the domain.

**Comment by Cabinetman** [comment]
This week I helped writing documentation for section 2.1.4 (Narration) and reviewed a classmate's contribution, providing constructive feedback on their work.

**Comment by elliot-gomez** [comment]
This week, I created multiple documentation tasks for the team to work on, targeting the completion of sections for the Milestone 1 documentation. I personally worked on and completed three of these tasks, each covering individual sections of the documentation: the current situation, needs, and ideas for the project.
- Elliot Julian Gomez Coriano

**Comment by Kaariinaa08** [comment]
For the third week of this milestone, I focused on reviewing and refining my previous two tasks to ensure they met the requirements. No new tasks were taken on this week.

- Karina Lopez Rodriguez

**Comment by cristian-barreras** [comment]
This week I worked on creating the interest forum and documentation. I completed the tasks of making the interest forum and filling out section 1.4 ( Other Activities Beyond Source Code Development) of documentation.

**Comment by franrov4** [comment]
Worked on the merging process of the database team's work with the main project. 

~Francisco Rovira Landron

**Comment by JadStelar** [comment]
Will be reviewing and proofreading the document for this milestone.
-Jahdiel Montero

**Comment by RyanVelez101** [comment]
This week I worked on the seller login and signup UI. 
~Ryan

**Comment by YampierFernandez** [comment]
Worked on documentation task for making the Domain rough Sketch

**Comment by FabiolaZTorres** [comment]
Implemented input validation guidelines for username, email, and phone number fields on the Sign-Up page, ensuring that only properly formatted data can be submitted. Also reviewed various tasks and added the Lead Reviewed tag to them for approval. 

-Fabiola Z. Torres Maldonado

**Comment by Christian2101** [comment]
Have been continuing to complete the documentation. Will be filling out the final parts getting it ready for before 8PM

**Comment by AlvinBadillo** [comment]
This week I worked with the creation of the item controller. I defined the following CRUD operations:
- createItem()
- getAllItems()
- getItemById()
These functions were created based on the item model implementation.
